akka-http-core/src/test/scala/akka/http/engine/parsing/HttpHeaderParserSpec.scala

```diff
@@ -153,7 +153,8 @@ class HttpHeaderParserSpec extends WordSpec with Matchers with BeforeAndAfterAll
           |   |   ┌─o-r-i-g-i-n-:- (origin)
           |   |   |                   ┌─e-n-t-i-c-a-t-e-:- (proxy-authenticate)
           |   | ┌─p-r-o-x-y---a-u-t-h-o-r-i-z-a-t-i-o-n-:- (proxy-authorization)
-          |   | | └─r-a-n-g-e-:- (range)
+          |   | | |   ┌─a-n-g-e-:- (range)
+          |   | | └─r-e-f-e-r-e-r-:- (referer)
```

Comment:
Yup. I'll keep them in but disable them.
They do serve the purpose of providing a nice test bed for the fast head parser / cache implementation, which does exhibit some size and structure restrictions.
However, since the code has proven quite resilient to growth in the header model  (i.e. keeps working as expected)
over the course of the last two years that we now have this code in, disabling these tests should be fine.